src/pymovements/gaze/gaze.py

```diff
@@ -1751,6 +1752,151 @@ def __repr__(self) -> str:
         """Return string representation of Gaze."""
         return self.__str__()
 
+    def save(
+            self,
+            dirname: str | Path,
+            save_events: bool = True,
```

Comment:
I'm wondering if we could change these arguments to `save_events: bool | None = None`.

This way we have three options:

- `save_events is None`: save the events if `Gaze.events is not None`
- `save_events == False`: don't save
- `save_events == True`: save the events, but raise an error if `gaze.events is None or len(gaze.events) == 0`

This could be nice for workflows, where you really want to make sure that you won't save empty files.